docs/changelog.md

```diff
@@ -46,6 +46,7 @@ Docs
 ----
 
 - Add docs for event handlers
+- Expanded "Setting up Environment" section to be more beginner friendly
```

Comment:
We typically don't add changelogs for docs changes - this has been a point of confusion, so I actually updated the PR template this morning. 😅

Reply:
Oops. I'll take it out then. 😅